posts/2024-06-04-24.0.0.6-beta.adoc

```diff
@@ -0,0 +1,455 @@
+
+|===
+
+You can define the `mp.metrics.distribution.percentiles` property similar to the following example.
+
+----
+mp.metrics.distribution.percentiles=alpha.timer=0.5,0.7,0.75,0.8;alpha.histogram=0.8,0.85,0.9,0.99;delta.*=
+----
+
+This property creates the `alpha.timer` timer metric to track and output the 50th, 70th, 75th, and 80th percentile values. The `alpha.histogram` histogram metric outputs the 80th, 85th, 90th, and 99th percentile values. Percentiles for any histogram or timer metric that matches with `delta.*` are disabled.
+
+We'll expand on this example and define histogram buckets for the `alpha.timer` timer metric by using the `mp.metrics.distribution.timer.buckets` property.
+
+----
+mp.metrics.distribution.timer.buckets=alpha.timer=100ms,200ms,1s
+----
+
+This configuration tells the metrics runtime to track and output the count of durations that fall within 0-100ms, 0-200ms and 0-1 seconds. This output is due to the histogram buckets working in a _cumulative_ fashion.
+
+The corresponding prometheus output for the `alpha.timer` metric at the `/metrics` REST endpoint is similar to the following example:
+
+----
+# TYPE application_alpha_timer_mean_seconds gauge
+application_alpha_timer_mean_seconds 2.9700022497975187
+# TYPE application_alpha_timer_max_seconds gauge
+application_alpha_timer_max_seconds 5.0
+# TYPE application_alpha_timer_min_seconds gauge
+application_alpha_timer_min_seconds 1.0
+# TYPE application_alpha_timer_stddev_seconds gauge
+application_alpha_timer_stddev_seconds 1.9997750210918204
+# TYPE alpha_timer_seconds histogram <1>
+application_alpha_timer_seconds_bucket{le="0.1"} 0.0 <2>
+application_alpha_timer_seconds_bucket{le="0.2"} 0.0 <2>
+application_alpha_timer_seconds_bucket{le="1.0"} 1.0 <2>
+application_alpha_timer_seconds_bucket{le="+Inf"} 2.0 <2> <3>
+application_alpha_timer_seconds_count 2
+application_alpha_timer_seconds_sum 6.0
+application_alpha_timer_seconds{quantile="0.5"} 1.0
+application_alpha_timer_seconds{quantile="0.7"} 5.0
+application_alpha_timer_seconds{quantile="0.75"} 5.0
+application_alpha_timer_seconds{quantile="0.8"} 5.0
+----
+
+<1> The Prometheus metric type is `histogram`. Both the quantiles/percentile and buckets are represented under this type.
+<2> The `le` tag represents _less than_ and is for the defined buckets, which are converted to seconds.
+<3> Prometheus requires that a `+Inf` bucket counts all hits.
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28337
+// Contact/Reviewer: anjumfatima90
+// // // // // // // //
+[#jcache]
+== InstantOn support distributed HTTP session caching

```

Comment:
Should this be `InstantOn support for distributed HTTP session caching` ?

Reply:
fixed here and in anchor list